components/MainView.js

```diff
@@ -89,6 +89,7 @@ window.onload = function () {
 
                 this.setState(() => ({ itemList: this.shuffle(inputList) }))
             }
+            this.setState({ displayText: `Press Enter to Begin\nPress E to Edit List (${this.state.itemList.length || 0})` })
```

Comment:
Same concern as above.

Reply:
Issue is if the item list is mutated to a value of null or the data is malformed in some way.

Reply:
The || 0 is just to catch any value that is evaluated to not be a numeric length. I can remove this check if you think it’s best?